docs/upgrade/v1-4-2-to-v1-5-0.md

```diff
@@ -136,3 +136,70 @@ To fix the issue, perform any of the following actions:
 
 Related issues:
 - [[BUG] AirGap Upgrades Seem Blocked with Fluentbit/FluentD](https://github.com/harvester/harvester/issues/7955)
+
+### 3. Upgrade stuck on waiting for Harvester bundle
+
+This occurs when you upgrade harvester from old vesrions, including `v1.0.x`, `v1.1.x` and `v1.2.x`.
+
+```shell
+> kubectl get bundles -n fleet-local
+NAME                                          BUNDLEDEPLOYMENTS-READY   STATUS
+mcc-harvester                                 0/1                       Modified(1) [Cluster fleet-local/local]; kubevirt.kubevirt.io harvester-system/kubevirt modified {"spec":{"configuration":{"vmStateStorageClass":"vmstate-persistence"}}}
+```
+
+The root cause is that we didn't apply the latest `depdendency_charts` CRDs because Helm doesn't manage CRDs for us. In order to make upgrade continue, please execute folloing script. It includs `kubevirt`, `csi-snapshotter` and `whereabouts`.
```

Comment:
```suggestion
The root cause is that we didn't apply the latest `depdendency_charts` CRDs because Helm doesn't manage CRDs for us. In order to make upgrade continue, please execute following script. It includes `kubevirt`, `csi-snapshotter` and `whereabouts`.
```